tests/test_config.py

```diff
@@ -224,6 +224,32 @@ def bob_config(transient_to, initial_config):
     assert "quit" not in bob.text()
 
 
+# Hircd currently doesn't support the WHOIS command
+# IrcCore._nickmask will therefore always be None in the test.
```

Comment:
This text should go into the `reason="..."` string, which is currently wrong as it was copy/pasted.